pkg/minishift/openshift/version/openshift_versions.go

```diff
@@ -35,7 +35,7 @@ import (
 	"github.com/minishift/minishift/pkg/util"
 )
 
-type ImageTags struct {
+type releaseTags struct {
```

Comment:
@anjannath There is no use of following `ImangeInfo` struct now also. L42-L50 

```golang
 type ImageInfo struct {
 	Size         int         `json:"size"`
 	Architecture string      `json:"architecture"`
 	Variant      interface{} `json:"variant"`
 	Features     interface{} `json:"features"`
 	Os           interface{} `json:"os"`
 	OsVersion    interface{} `json:"os_version"`
 	OsFeatures   interface{} `json:"os_features"`
 }
```

Reply:
Aha.. Removing it, shouldn't i have got an error while compiling..?  :-/